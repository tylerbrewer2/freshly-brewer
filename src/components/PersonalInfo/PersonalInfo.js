import React from 'react';

import Col from '../Col';
import DescriptionText from '../DescriptionText';
import InitialsSymbol from '../InitialsSymbol';
import Row from '../Row';
import Section from '../Section';

const PersonalInfoCore = () => {
  return (
    <Section>
      <Row>
        <Col tabletWidth={5} mobileWidth={12}>
          <DescriptionText
            header="MORGAN LINDSAY PARSONS"
            subHeader="THE BRIDE"
            align="right"
            mobileAlign="left"
          >
            Morgan is the owner and founder of the (future) $1,000,000+ company Morgan Parsons Creative. She is perfectly content with potatoes for breakfast, lunch and dinner and
            is the 2011 Chick-fil-a Cow of the Year. Morgan is caring and always thinking about others, and I can't wait to get to know more and more every day. She's also hilarious and has a firey passion of saying "How bout dah" on a daily basis.
          </DescriptionText>
        </Col>
        <Col mobileWidth={0} tabletWidth={2}>
          <InitialsSymbol />
        </Col>
        <Col tabletWidth={5} mobileWidth={12}>
          <DescriptionText
            header="TYLER CLARK BREWER"
            subHeader="THE GROOM"
          >
            Most days you’ll find Tyler biking through the streets of Washington D.C., saving the world with code or ferociously chowing down on a Chipotle burrito the size of his head. I think I knew he was the one for me the day he asked me out… with a Chickfila sandwich. ;) #blessed.  In all seriousness, Tyler is passionate, hard working and the kindest person I know and I am excited to do life with him.
          </DescriptionText>
        </Col>
      </Row>
    </Section>
  )
}

export default PersonalInfoCore;