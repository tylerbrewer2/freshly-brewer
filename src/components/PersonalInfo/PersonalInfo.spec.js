import React from 'react';
import renderer from 'react-test-renderer';

import PersonalInfo from './PersonalInfo';

it('renders correctly', () => {
  const tree = renderer.create(
    <PersonalInfo />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
