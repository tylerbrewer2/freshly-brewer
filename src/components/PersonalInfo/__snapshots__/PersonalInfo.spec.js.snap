exports[`test renders correctly 1`] = `
<section
  className="sc-EHOje gVlGrG">
  <div
    className="undefined   row">
    <div
      className="col-xs-12 
          col-sm-5 
          undefined">
      <div
        className="sc-htpNat fqYUEc">
        <h1
          className="sc-bdVaJa kgyTWr">
          MORGAN LINDSAY PARSONS
        </h1>
        <h1
          className="sc-bdVaJa kgyTWr">
          THE BRIDE
        </h1>
        <p
          className="sc-bwzfXH jCzhFF">
          Morgan is the owner and founder of the (future) $1,000,000+ company Morgan Parsons Creative. She is perfectly content with potatoes for breakfast, lunch and dinner and is the 2011 Chick-fil-a Cow of the Year. Morgan is caring and always thinking about others, and I can\'t wait to get to know more and more every day. She\'s also hilarious and has a firey passion of saying \"How bout dah\" on a daily basis.
        </p>
      </div>
    </div>
    <div
      className="col-xs-undefined 
          col-sm-2 
          undefined">
      <div
        className="sc-ifAKCX jPHgwG">
        <div
          className="sc-bxivhb hhnUjy"
          color="black">
          M
        </div>
        <div
          className="sc-bxivhb hhnUjy"
          color="black">
          x
        </div>
        <div
          className="sc-bxivhb hhnUjy"
          color="black">
          T
        </div>
      </div>
    </div>
    <div
      className="col-xs-12 
          col-sm-5 
          undefined">
      <div
        className="sc-htpNat exbpEc">
        <h1
          className="sc-bdVaJa kgyTWr">
          TYLER CLARK BREWER
        </h1>
        <h1
          className="sc-bdVaJa kgyTWr">
          THE GROOM
        </h1>
        <p
          className="sc-bwzfXH jCzhFF">
          Most days you’ll find Tyler biking through the streets of Washington D.C., saving the world with code or ferociously chowing down on a Chipotle burrito the size of his head. I think I knew he was the one for me the day he asked me out… with a Chickfila sandwich. ;) #blessed.  In all seriousness, Tyler is passionate, hard working and the kindest person I know and I am excited to do life with him.
        </p>
      </div>
    </div>
  </div>
</section>
`;
